app/assets/stylesheets/application.css.scss

```diff
@@ -159,3 +159,8 @@ main {
   max-width: 150%;
   margin-left: 0px;
 }
+
+.avatar{
+  background-color: black;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.